packages/gamut-labs/src/landingPage/PageVideoGallery.tsx

```diff
@@ -0,0 +1,63 @@
+import { Video } from '@codecademy/gamut';
+import { breakpoints } from '@codecademy/gamut-styles';
+import styled from '@emotion/styled';
+import React from 'react';
+
+const Grid = styled.div`
+  display: grid;
+  gap: 32px;
+
+  // there must be a mathematical explanation for why ~12:5 is the correct ratio for
+  // the column widths but I do not know what it is. Regardless, the effect is that
+  // all three videos will maintain a 16:9 aspect ratio with an even gap between them.
+  grid-template: 1fr 1fr / 12fr 5fr;
+
+  @media only screen and (max-width: ${breakpoints.lg}) {
```

Comment:
Oh, and we have some utils for these breakpoints:

```suggestion
  ${mediaQueries.lg} {
```